example/android/app/build.gradle

```diff
@@ -204,7 +204,7 @@ dependencies {
     implementation "com.facebook.react:react-native:+"  // From node_modules
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
     implementation 'com.google.android.material:material:1.4.0'
-    implementation 'com.stripe:stripe-wechatpay:17.1.0'
+    // implementation 'com.stripe:stripe-wechatpay:17.1.0'
```

Comment:
This change will be removed, just need this temporarily to build correctly if testing locally.